src/execution_tree/primitives/assert_condition.cpp

```diff
@@ -37,7 +37,13 @@ namespace phylanx { namespace execution_tree { namespace primitives
     {
         hpx::util::make_tuple("assert",
             std::vector<std::string>{"assert(_1)"},
-            &create_assert_condition, &create_primitive<assert_condition>)
+            &create_assert_condition, &create_primitive<assert_condition>,
+            "cond\n"
+            "Args:\n"
+            "\n"
+            "    cond (boolean expression) : if not true, raise an AssertionError\n"
+            "Returns:\n"
+            )
```

Comment:
Have you considered using raw strings for the documentation string? This would remove the need to explicitly place `\n` and similar in the strings. For instance the above snippet would read (similar to pythons `'''` string notation):
```
        hpx::util::make_tuple("assert", 
            std::vector<std::string>{"assert(_1)"}, 
            &create_assert_condition, &create_primitive<assert_condition>) 
            R"(
cond 
Args:
    cond (boolean expression) : if not true, raise an AssertionError
Returns: 
   <nothing>
)"
```

Reply:
I didn't know about them.